.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,76 @@
+# https://help.github.com/en/categories/automating-your-workflow-with-github-actions
```

Comment:
Let me just add a comment on rootlevel. Isnt it maybe wise to integrate all tooling already that we want to have implemented? Like PHPstan and maybe some comp. checks or will we add them later

Reply:
@pimjansen 

I'd prefer to add them later, one by one.